### ISSUE TITLE
can't get the acc1 76.x as shown

### ISSUE THREAD

**Issue by xiaocaijizzz** [fastalgo/imagenet_resnet50_lamb#1]
I have followed your argument set:
bs=32768
lr=0.04
warm_up epoch=20
train epoch=90
weight_decay=0
weight_decay_input=1.5
lr_schedule=poly and poly_power=1 (I have read the code, and find that actually a linear decay is used in  training-lr-schedule,  that is equal to poly schedule with power=1.0)
After training, I get top1 accutacy as follows:
32k: 72.052
16k: 75.54
8k: 74.584
4k: 75.94
when training bs=16k/8k/4k，I only change the lr and warm-up epoch as the article.
All training are conducted in pyTorch, GPU. 
Is there any thing else I haven't noticed? Can anyone who have reproduced the accuracy of the paper can give me some suggesstions?

**Comment by fastalgo** [comment]
For PyTorch/GPU version, try this:

https://github.com/NUS-HPC-AI-Lab/pytorch-lamb

Please let me know if you still can't reproduce it.

**Comment by xiaocaijizzz** [comment]
> For PyTorch/GPU version, try this:
> 
> https://github.com/NUS-HPC-AI-Lab/pytorch-lamb
> 
> Please let me know if you still can't reproduce it.

**Comment by xiaocaijizzz** [comment]
> For PyTorch/GPU version, try this:
> 
> https://github.com/NUS-HPC-AI-Lab/pytorch-lamb
> 
> Please let me know if you still can't reproduce it.

thank you! I will try it and reply to you the update

**Comment by xiaocaijizzz** [comment]
> For PyTorch/GPU version, try this:
> 
> https://github.com/NUS-HPC-AI-Lab/pytorch-lamb
> 
> Please let me know if you still can't reproduce it.
